pkg/leakybucket/processor.go

```diff
@@ -21,8 +21,8 @@ func (d *DumbProcessor) OnBucketPour(b *BucketFactory) func(types.Event, *Leaky)
 	}
 }
 
-func (d *DumbProcessor) OnBucketOverflow(b *BucketFactory) func(*Leaky, types.SignalOccurence, *Queue) (types.SignalOccurence, *Queue) {
-	return func(l *Leaky, s types.SignalOccurence, q *Queue) (types.SignalOccurence, *Queue) {
+func (d *DumbProcessor) OnBucketOverflow(b *BucketFactory) func(*Leaky, types.Alert, *Queue) (types.Alert, *Queue) {
+	return func(l *Leaky, s types.Alert, q *Queue) (types.Alert, *Queue) {
 		return s, q
```

Comment:
can you please rename the ones letter variable by more understandable name please ?